Composer/ScriptHandler.php

```diff
@@ -326,7 +326,7 @@ protected static function executeCommand(CommandEvent $event, $consoleDir, $cmd,
         $process = new Process($php.($phpArgs ? ' '.$phpArgs : '').' '.$console.' '.$cmd, null, null, null, $timeout);
         $process->run(function ($type, $buffer) use ($event) { $event->getIO()->write($buffer, false); });
         if (!$process->isSuccessful()) {
-            throw new \RuntimeException(sprintf('An error occurred when executing the "%s" command.', escapeshellarg($cmd)));
+            throw new \RuntimeException(sprintf('An error occurred when executing the "%s" command: %s.', escapeshellarg($cmd), $process->getErrorOutput()));
```

Comment:
I would output the error output of the executed process on a new line. Otherwise, this may look a bit weird when the produced output of the subprocess contains newlines itself.